[i18n-CN] Translated some files to implified Chinese  #1915

2404589803 opened the pull request:
Related to https://github.com/huggingface/huggingface_hub/issues/1915.

What is this PR about ?

I translated the following files to implified Chinese :

- toctree.yml
- index.md
- installation.md
- quick-start.md
- git_vs_http.md
- repository.md

What should you check ?

- Grammar and orthograph
- If my pull requests follows GitHub/open source contribution good practices (I am a beginner)

Wauplin commented on README.md:
```diff
@@ -21,7 +21,8 @@
         <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_hi.md">हिंदी</a> | 
-        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_ko.md">한국어</a>
+        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_ko.md">한국어</a> |
+         <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_cn.md">中文（简体）</a> 
```
Nice to add a `_cn` README! Could you also add the link to other READMEs? ([README_de.md](https://github.com/huggingface/huggingface_hub/blob/main/README_de.md), [README_ko.md](https://github.com/huggingface/huggingface_hub/blob/main/README_ko.md) and [README_hi.md](https://github.com/huggingface/huggingface_hub/blob/main/README_hi.md)). Thanks in advance!

2404589803 replied:
No problem, I will add it when I have some free time🦾. Thanks!@ Wauplin

2404589803 replied:
Chinese readme links have been added to readme files in other languages @Wauplin

Wauplin commented on docs/source/cn/_redirects.yml:
```diff
@@ -0,0 +1,11 @@
+# Move "how-to" pages to the guides/ folder
+how-to-cache: guides/manage-cache
+how-to-discussions-and-pull-requests: guides/community
+how-to-downstream: guides/download
+how-to-inference: guides/inference
+how-to-manage: guides/repository
+how-to-model-cards: guides/model-cards
+how-to-upstream: guides/upload
+search-the-hub: guides/search
+guides/manage_spaces: guides/manage-spaces
+package_reference/inference_api: package_reference/inference_client
```
This file is used in the English version of the docs to make legacy urls still compatible with the current documentation (when we renamed some files). So for simplified Chinese it is not necessary to have it given it is a brand new documentation.

2404589803 replied:
File deleted @Wauplin

Wauplin commented on docs/source/cn/安装.md:
```diff
@@ -0,0 +1,142 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
```
Could you rename this file to its English name please? While the content must be in the target language (simplified Chinese) for the users, it is still better to have the file architecture in English so that the maintainers know which file contains what. Also, the `_toctree.yml` file is expecting English-based files (see `local: index` means "take the `index.md` local file).

This apply to other translated files and directories. Thanks in advance for making the changes!

2404589803 replied:
No problem, I'll modify it when I have some free time 🦾. Thanks! @Wauplin

2404589803 replied:
File name has been renamed to English @Wauplin

Wauplin reviewed (commented):
Hi @2404589803! Thanks for opening this PR! I am very enthusiastic of having this documentation translated to (simplified) Chinese! I have added a few technical comments about how to structure the PR but will not be able to review the content myself :slightly_smiling_face: Do you know people from the Chinese-speaking community that would be interested in reviewing your work?

Wauplin commented:
Ping @adeenayakup and @xianbaoqian who will be interested in this! :hugs:

Wauplin reviewed (commented):
Another comment I forgot: you should update the github workflow files so that `cn` docs are built as well. You should add `cn` in the languages parameter [here](https://github.com/huggingface/huggingface_hub/blob/7319ea4b1df8f0d025dec97bccf5255ba4970d45/.github/workflows/build_documentation.yaml#L16) and [here](https://github.com/huggingface/huggingface_hub/blob/7319ea4b1df8f0d025dec97bccf5255ba4970d45/.github/workflows/build_pr_documentation.yaml#L17).

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/huggingface_hub/pr_1916). All of your documentation changes will be reflected on that endpoint. The docs are available until 30 days after the last update.

xianbaoqian commented on docs/source/cn/_toctree.yml:
```diff
@@ -0,0 +1,74 @@
+      title: 快速开始
+    - local: installation
+      title: 安装
+- title: "How-to guides"  #"操作指南"
```
I think that we do need to translate the title so that they can show up in Chinese.

2404589803 commented on docs/source/cn/index（索引）.md:
Sorry, this is a mistake in my work, I will correct it as soon as possible，thanks @ xianbaoqian

2404589803 replied:
The file name has been changed back to English @xianbaoqian

2404589803 commented on docs/source/cn/installation（安装）.md:
```diff
@@ -0,0 +1,199 @@
+
+### Editable install  
+
+Installing from source allows you to setup an [editable install](https://pip.pypa.io/en/stable/topics/local-project-installs/#editable-installs).
```
Sorry, this is a mistake in my work, I will correct it as soon as possible，thanks @ xianbaoqian

adeenayakup reviewed (commented):
Hey @2404589803. Your contribution is very much appreciated! 🤗 Here are some suggestions for the translation, don't hesitate to drop me a message if needed.

2404589803 commented on docs/source/cn/索引.md:
```diff
@@ -0,0 +1,43 @@
+
+# 🤗 Hub 客户端库
+
+`huggingface_hub`库允许您与[Hugging Face Hub](https://hf.co)进行交互，这是一个面向创作者和合作者的机器学习平台。您可以为您的项目发现预训练模型和数据集，或试用 Hub 上托管的数百个机器学习应用程序。您还可以使用 Python 创建和共享您自己的模型和数据集。 `huggingface_hub`库为您提供了一种简单的方法来执行所有这些操作。
```
Thank you for your suggestion, I will modify it as soon as possible @adeenayakup

Wauplin commented:
Thanks a lot for making all the requested changes so quickly @2404589803! :pray: From a technical point of view, this PR is good and ready to be merged. From a content point of you (the translated parts), I let you iterate with @xianbaoqian and @adeenayakup 's feedback. Let me know once it's done!

2404589803 commented:
OK, please wait until I finish the iteration before merging the branches, thank you very much ！@Wauplin

2404589803 commented:
At the same time, I am very grateful to everyone for your constructive suggestions on my work. Thank you.🤗 @Wauplin @xianbaoqian @adeenayakup

2404589803 commented:
I have corrected all the above changes, and I also changed the link pointing inside the translated file (this link is used to jump between project files), because if there is no change, it will appear after merging the branch problem, although the review did not point this out.@Wauplin

xianbaoqian commented on docs/source/cn/installation.md:
````diff
@@ -0,0 +1,142 @@
+.env/Scripts/activate 
+```
+
+现在您可以从[从 PyPi 注册表 ](https://pypi.org/project/huggingface-hub/)安装 `huggingface_hub`：
````
```suggestion
现在您可以从[从 PyPi 注册表](https://pypi.org/project/huggingface-hub/)安装 `huggingface_hub`：
```

xianbaoqian commented on README.md:
```diff
@@ -21,7 +21,8 @@
         <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_hi.md">हिंदी</a> | 
-        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_ko.md">한국어</a>
+        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_ko.md">한국어</a> |
+         <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_cn.md">中文（简体）</a> 
```
```suggestion
        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_cn.md">中文（简体）</a> 
```

Wauplin reviewed (approved):
So cool! Love the quick feedback loop. Thanks again everyone for working on this piece. From my point of view it's good to go to let's merge it! :rocket:
